components/calendar/CalDescription.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <div class="desc-size overflow-wrap scroll">
+    <span v-html="handledDesc"></span>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'Description',
+  props: {
+    /**
+     * The HTML of the event description.
+     */
+    desc: { type: String, required: true },
+    /**
+     * The URL to more info on the event.
+     */
+    url: String
+  },
+  computed: {
+    /**
+     * Parses and formats the description. If the raw description contains a Zoom link, then the method
+     * returns just that link. Otherwise, trims the description and appends a link to more information.
+     * @returns {string} The formatted description.
+     */
+    handledDesc() {
+      if (!this.desc) return '';
+      if (this.desc.includes('is inviting you to a scheduled Zoom meeting.')) {
+        const zoomLink = this.desc.match(
+          'https://brown.zoom.us/[a-zA-Z0-9/@:%._+~#?&/=]*'
+        )[0];
+        return (
+          '<p>' +
+          '<p>' +
+          'Zoom link: <br><br>' +
+          "<a href='" +
+          zoomLink +
+          "'>" +
+          zoomLink +
+          '</a>' +
+          '</p>'
+        );
+      } else {
+        return (
+          this.trimDesc(this.desc) +
+          '...' +
```

Comment:
I'd add the `...` to `trimDesc` instead of here so that we don't get a ... on an short, but whole description